src/ResourceManager.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include <unordered_map> //HashMap
+#include <SFML/Graphics.hpp>
+#include <SFML/Audio.hpp>
+
+class ResourceManager;
+
+class ResourceManager {
+private:
+	ResourceManager(); //constructor
+
+	void LoadTexture(const char* name, const char* path);
+	void LoadSound(const char* name, const char* path);
+	void LoadFont(const char* name, const char* path);
+
+	std::unordered_map<const char*, sf::Texture> TextureMap;
+	std::unordered_map<const char*, sf::Font> FontMap;
+	std::unordered_map<const char*, sf::Sound> SoundMap;
```

Comment:
As per comment in .cpp file, we don't need a map for that. ResourceManager should return SoundBuffers, not Sounds